File: beginner/c/1070.c
#include <stdio.h>
int main(){

    int n, i, cont = 0;

    scanf("%d", &n);

    for(i = n; i > 0; i++){
        if(cont == 6){
            break;
        }
        else if(i % 2 != 0){
            printf("%d\n", i);
            cont++;
        }
    }


    return 0;
    
}

File: beginner/javascript/1070.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');
var n, i, cont = 0;

n = parseInt(valores.shift());

for(i = n; i > 0; i++){
    if(cont == 6){
        break;
    }
    else if(i % 2 != 0){
        console.log(i);
        cont++;
    }
}